vms/shift/services.py

```diff
@@ -21,6 +22,7 @@ def add_shift_hours(v_id, s_id, start_time, end_time):
     if volunteer_shift:
         volunteer_shift.start_time = start_time
         volunteer_shift.end_time = end_time
+        volunteer_shift.date_logged = timezone.now()+timedelta(days=7)

```

Comment:
Please don't do it like this. Let it be `volunteer_shift.date_logged = timezone.now()` and handle the 7 days constraint somewhere else. This way, things will be difficult for the other contributors who will be working after you on the same code.